### DISCUSSION TITLE
Admin commands output

### DISCUSSION THREAD

**Discussion by dgadelha** [piqueserver/piqueserver#643] [Feature Requests]
One of the things I asked myself when using PySnip was if there was really a need of showing to everyone what an admin is doing (commands).

For some commands, like togglebuild, toggleteamkill, there is really a need of noticing players. But using goto, tp, god (even though there's a silent for these), kill, heal, it would be okay to show only to the other admins, but showing to everyone is a bit unneeded.

**Comment by NotAFile** [comment]
The actual admin notification part is a hard decision.

Showing messages prevents admin abuse and for most commands, there is just no reason to run them silently if not cheating, e.g. /heal. I fear that if these commands were left silent, admins would abuse them. So i'd disagree, there *is* a need to show that to the player: to know if the admins are playing fair or not. I understand it can be annoying though. Is there any scenario in particular where you wanted a silent command but it didn't exist? If it's reasonable we might add something like that.

**Comment by dgadelha** [comment]
Alright, created #124 and #125.

About the admin commands, well, I wish that the admin could just manage the server without letting everyone know what they're doing. I know that this could lead into cheating, but well, I wouldn't make someone who's a bad admin, an admin.

There could be at least an option for that.
